test/combineReducers.spec.js

```diff
@@ -18,14 +18,30 @@ describe('Utils', () => {
       expect(s2).toEqual({ counter: 1, stack: [ 'a' ] })
     })
 
-    it('ignores all props which are not a function', () => {
+    it('warns on props which are not a function and excludes them', () => {
+      const preSpy = console.error
+      const spy = jest.fn()
+      console.error = spy
```

Comment:
You've overridden `console.error`, but there's no mechanism here to put it back after the test.

The Jest way to do this is to make a mock of `utils/warning` to capture any calls to `warning()`.

```js
jest.mock('../src/helpers/warning')
```

https://facebook.github.io/jest/docs/en/manual-mocks.html has more info :)

Reply:
They can do what we do in other tests. Just add this at the end:
```js
       spy.mockClear()
       console.error = preSpy
```